### PR TITLE
Fix multifield querying

### PR DESCRIPTION
**Pull request by vyasswaroop**
- [x] Allow request if any queryable field is present
- [x] Update tests
- [x] Enrich
- [x] Resolve
- [ ] Update version to 1.1.1

### CODE REVIEW

**Comment by vyasswaroop** [comment]
[ch128325]

File: fc/fullcontact.go [hunk 1]
```diff
@@ -276,8 +282,12 @@ func (fcClient *fullContactClient) IdentityDelete(resolveRequest *ResolveRequest
 }
 
 func (fcClient *fullContactClient) resolveRequest(ch chan *APIResponse, resolveRequest *ResolveRequest, url string) chan *APIResponse {
+	err := validateResolveRequest(resolveRequest)
```
Comment (prakhar-j) [inline fc/fullcontact.go:285]: We dont need this validation for `resolve` and `delete`, so we can just call this inside `validateForIdentityMap` in `resolve_request.go`

**Review by prakhar-j — CHANGES_REQUESTED** [review]
Thanks for updating tests. I didn't see any version update. If you are planning to bump version, please update in `constants.go` and then add corresponding tag after merging

File: fc/fullcontact.go (hunk 1 above)
Comment (vyasswaroop) [reply fc/fullcontact.go:285]: I didn't get it. I guess if we dont use the `validateResolveRequest` here, then there is no reference to this validation at all anywhere. So should we remove the `validateResolveRequest` section itself?

Comment (prakhar-j) [reply fc/fullcontact.go:285]: We can call it inside https://github.com/fullcontact/fullcontact-go/blob/master/fc/resolve_request.go#L57
as we need to only validate mfReq in `identity.map`

**Review by prakhar-j — APPROVED** [review]

**Pull request merged**